Realm/RLMRealm.mm

```diff
@@ -447,6 +447,12 @@ + (instancetype)realmWithConfiguration:(RLMRealmConfiguration *)configuration
         }
     }
 
+    if (configuration.seedFilePath && ![RLMRealm fileExistsForConfiguration:configuration]) {
```

Comment:
This is a race condition if multiple processes hit this. We need to do this inside a call to `DB::call_with_lock()` to ensure that only one process tries to copy the file, and that the second process doesn't open the file while the copy is in progress.